app/src/components/blocks/_lloydGeorge/lloydGeorgeViewRecordStage/LloydGeorgeViewRecordStage.tsx

```diff
@@ -212,6 +212,10 @@ function LloydGeorgeViewRecordStage({
 
             <h1>{pageHeader}</h1>
             <PatientSimpleSummary />
+            <p>
+                To search within the record use <strong>Control</strong> and <strong>F</strong>
```

Comment:
Minor nitpick but the language in the ticket says "this record" and not "the record"